### PR TITLE
Branch ab

### PR DESCRIPTION
**Pull request by BinaryHexedecimal**

### CODE REVIEW

File: DIKUArcade/Entities/Entity.cs [hunk 1]
```diff
@@ -30,11 +30,13 @@ public class Entity {
         }
 
         public void RenderEntity() {
-            Image.Render(Shape);
+            if(!isDeleted){
+            Image.Render(Shape);}
```
Comment (oleks) [inline DIKUArcade/Entities/Entity.cs:34]: This is really weird bracing. I think you want to use an auto-formatter.

File: DIKUArcade/Entities/Entity.cs [hunk 2]
```diff
@@ -30,11 +30,13 @@ public class Entity {
...
         }
 
         public void RenderEntity(Camera camera) {
-            Image.Render(Shape, camera);
+            if(!isDeleted){
+            Image.Render(Shape, camera);}
```
Comment (oleks) [inline DIKUArcade/Entities/Entity.cs:39]: Similarly here.

File: DIKUArcade/GUI/Window.cs [hunk 3]
```diff
@@ -9,10 +9,14 @@
 using OpenTK.Graphics.OpenGL;
 using DIKUArcade.Input;
 
+using DIKUArcade.Events;
+
 namespace DIKUArcade.GUI {
     /// <summary>
     /// This class represents a graphical window in the DIKUArcade game engine.
     /// </summary>
+   // public class Window: IGameEventProcessor {
```
Comment (oleks) [inline DIKUArcade/GUI/Window.cs:18]: Why retain the comment? If it is to retain a trace that perhaps `Window` should implement `IGameEvenProcessor`, then I would write this as:

```csharp
public class Window { // : IGameEventProcessor
```

File: DIKUArcade/GUI/Window.cs [hunk 4]
```diff
@@ -373,5 +377,6 @@ public class Window {
             bmp.Save(path);
             Console.WriteLine($"Screenshot saved as: {path}");
         }
+
```
Comment (oleks) [inline DIKUArcade/GUI/Window.cs:380]: It is rather unusual to place an empty line like this.

File: DIKUArcade/Graphics/IBaseImage.cs [hunk 5]
```diff
@@ -4,5 +4,6 @@ namespace DIKUArcade.Graphics {
     public interface IBaseImage {
         void Render(Shape shape);
         void Render(Shape shape, Camera camera);
+
```
Comment (oleks) [inline DIKUArcade/Graphics/IBaseImage.cs:7]: Also not needed.

File: DIKUArcade/Graphics/Image.cs [hunk 6]
```diff
@@ -23,6 +23,5 @@ public class Image : IBaseImage {
 
         public Texture GetTexture() {
             return texture;
-        }
     }
-}
\ No newline at end of file
+    }}
```
Comment (oleks) [inline DIKUArcade/Graphics/Image.cs:27]: Please use an auto-formatter.

File: DIKUArcade/DIKUGame.cs [hunk 7]
```diff
@@ -16,7 +16,13 @@ public abstract class DIKUGame {
         /// </summary>
         public static int Timestep { get; private set; }
 
+        //[DllImport("libglfw.so.3")] // On Linux
+        //public static extern int glfwInit();
+
+
         public DIKUGame(WindowArgs windowArgs) {
+
+
```
Comment (oleks) [inline DIKUArcade/DIKUGame.cs:25]: What is the purpose of these line breaks?

File: DIKUArcade/DIKUArcade.csproj [hunk 8]
```diff
@@ -5,7 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="OpenTK" Version="4.5.0" />
+    <PackageReference Include="OpenTK" Version="4.1.0" />
```
Comment (oleks) [inline DIKUArcade/DIKUArcade.csproj:8]: Why downgrade?

**Comment by oleks** [comment]
What are the motivations for these changes?

**Review by oleks — COMMENTED** [review]
Please remember to provide context with your changes. Use proper commit messages, explain your changes. It might be clear to you (and someone else) right now. However, it will be indistinguishable from magic 6 months from now.

**Comment by BinaryHexedecimal** [comment]
omg
Sorry, it was just a mistake…I did not mean it.
Please just neglect my change request…..

**Pull request closed without merging**